build/angular-slider.js

```diff
@@ -1395,7 +1410,10 @@ angular.module('vr.directives.slider', ['ngTouch']).directive('slider',
                                  */
                                 function redrawSliders() {
                                     setPointers();
-                                    adjustBubbles();
+                                    // Only adjust bubbles if allowed
+                                    if(scope.noBubbleAdjustment === undefined) {
```

Comment:
Yes, but I also know that Greg preferred writing `if()` when he was at Incuna, even though Incuna's style guide says `if ()` =)